src/main/java/me/botsko/prism/actions/entity/PrismItemStack.java

```diff
@@ -0,0 +1,44 @@
+package me.botsko.prism.actions.entity;
+
+import com.google.common.base.Preconditions;
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.Serializable;
```

Comment:
Codacy found an issue: [Import statement for 'java.io.Serializable' is in the wrong order. Should be in the 'STANDARD_JAVA_PACKAGE' group, expecting group 'SPECIAL_IMPORTS' on this line.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=6368169)